src/main/java/buildcraftAdditions/client/gui/gui/GuiFluidicCompressor.java

```diff
@@ -85,8 +68,51 @@ public String getInventoryName() {
 	}
 
 	@Override
+	public void widgetActionPerformed(WidgetBase widget) {
+		if (widget instanceof WidgetButtonFluidicCompressor) {
+			fluidicCompressor.fill = ((WidgetButtonFluidicCompressor) widget).fill;
+			PacketHandler.instance.sendToServer(new MessageFluidicCompressorA(fluidicCompressor.fill, fluidicCompressor));
```

Comment:
Sorry, I missunderstood AEnterprises comment. 
I dont know how the other messages work, so i used the one that is already there. I did not refractor the GUI Code :P
Will look into it later this evening.


> Am 30.01.2015 um 17:52 schrieb BigXplosion <notifications@github.com>:
> 
> In src/main/java/buildcraftAdditions/client/gui/gui/GuiFluidicCompressor.java:
> 
> > @@ -85,8 +68,51 @@ public String getInventoryName() {
> >  	}
> >  
> >  	@Override
> > +	public void widgetActionPerformed(WidgetBase widget) {
> > +		if (widget instanceof WidgetButtonFluidicCompressor) {
> > +			fluidicCompressor.fill = ((WidgetButtonFluidicCompressor) widget).fill;
> > +			PacketHandler.instance.sendToServer(new MessageFluidicCompressorA(fluidicCompressor.fill, fluidicCompressor));
> You can use a MessageWidgetUpdate for this, and remove this packet
> 
> —
> Reply to this email directly or view it on GitHub.
>